Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.Data.cs

```diff
@@ -19,20 +20,20 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel
     /// </summary>
     public partial class WrapPanel
     {
-        internal class UvMeasure
+        [System.Diagnostics.DebuggerDisplay("U = {U} V = {V}")]
+        private struct UvMeasure
         {
-            internal double U { get; set; }
+            private const double FACTOR = 10000;
 
-            internal double V { get; set; }
+            private double u, v;
+            internal double U { get { return u ; } set { u = Math.Floor(value * FACTOR) / FACTOR; } }
 
-            public UvMeasure()
-            {
-                U = 0.0;
-                V = 0.0;
-            }
+            internal double V { get { return v ; } set { v = Math.Floor(value * FACTOR) / FACTOR; } }
```

Comment:
You don't need the FACTOR in this case mate :) right ? just do Math.Floor(value) with no multiplication or division.